src/sections/common/KnownDrugs/Body.js

```diff
@@ -203,7 +204,7 @@ function Body({
 
   const getWholeDataset = useCursorBatchDownloader(
     BODY_QUERY,
-    variables,
+    (variables = { ...variables, freeTextQuery: globalFilter }),
```

Comment:
I think we can just pass `{...variables, freeTextQuery: globalFilter}` as an argument without having to reassign `variables`

Reply:
Thanks @mirandaio good catch, change addressed 👍